pkg/env/core.go

```diff
@@ -48,7 +48,7 @@ func NewCore(ctx context.Context, log *logrus.Entry) (Core, error) {
 	if err != nil {
 		return nil, err
 	}
-	log.Infof("running on %s", im.Environment().Name)
+	log.Infof("InstanceMetadata: environment=%s subscriptionID=%s tenantID=%s location=%s resourceGroup=%s hostname=%s", im.Environment().Name, im.SubscriptionID(), im.TenantID(), im.Location(), im.ResourceGroup(), im.Hostname())
```

Comment:
_I think_ each log line in prod will have most of this information (except for cloud name) injected by loggin agent as a field. We definitelly already can search logs by location. I would probably just keep existing log line with environment name.